src/docs/30_mui_components/buttons.mdx

```diff
@@ -124,6 +124,13 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 		<LensToggle color="info" />
 		<LensToggle color="success" />
 		<LensToggle color="warning" />
-		<LensToggle color="error" />
+	</Box>
```

Comment:
The hover color for the first LensToggle's unselected option should be `bg.strong` instead of `bg.accent`